composer.json

```diff
@@ -11,8 +11,8 @@
     ],
     "require": {
         "php": ">=5.3.0",
-        "symfony/http-foundation": "~2.1",
-        "symfony/http-kernel": "~2.1"
+        "symfony/http-foundation": "~2.1 | ~3.0",
+        "symfony/http-kernel": "~2.1 | ~3.0"
```

Comment:
I would update to "~2.7|~3.0" as the other versions are not supported anymore. Also, we need to be sure that this works on both 2.7 and 3.0, so I would also change the `.travis.yml` file to be sure that we are running tests for at least those two versions. You can have a look at twigphp/twig for an example of how to make it work.